de.json

```diff
@@ -174,10 +174,10 @@
             }
         },
         "shop" : {
-            "title" : "Aktuelle Rotation",
+            "title" : "Aktueller Shop",

```

Comment:
What's the reason for this change? We don't want to deviate from the English phrase for no apparent reason if the German translation is understandable and precise.